chat-components/src/components/prechatsurveypane/PreChatSurveyPane.tsx

```diff
@@ -75,6 +75,8 @@ function PreChatSurveyPane(props: IPreChatSurveyPaneProps) {
                 font-size: ${props.styleProps?.customTextStyleProps?.fontSize} !important;
                 height: ${props.styleProps?.customTextStyleProps?.height};
                 padding-top: ${props.styleProps?.customTextStyleProps?.paddingTop};
+                overflow-wrap: break-word;
+                white-space: normal !important;
```

Comment:
Can we avoid using Important ? Like may be defining css to more deeper level ?

Reply:
AdaptiveCard has its own css defined at the same level. I don't see another way if we have to keep v1 parity

Reply:
AC has it's own css at the same level, which is defined later down the logic chain (which will always overwrite our styles). Technically we can add more parent selectors to remove the "!important", but that adds more dependencies (say a parent class name changes, we need to remember to change it here too, also more importantly the parent class names are all customizable). 

I think given that we need to keep V1 parity on this, using !important can be an acceptable exception